src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js

```diff
@@ -3,32 +3,32 @@ import { shallow } from 'enzyme';
 import InlineError from 'box-react-ui/lib/components/inline-error/InlineError';
 import ItemProperties from 'box-react-ui/lib/features/item-details/ItemProperties';
 import SidebarFileProperties, { SidebarFilePropertiesComponent } from '../SidebarFileProperties';
+import { KEY_CLASSIFICATION, KEY_CLASSIFICATION_TYPE } from '../../../constants';
 
 describe('components/ContentSidebar/SidebarFileProperties', () => {
     const getWrapper = (props) => shallow(<SidebarFilePropertiesComponent {...props} />);
-
-    test('should render the versions when total_count > 0', () => {
-        const props = {
-            file: {
-                created_at: 'foo',
-                description: 'foo',
-                modified_at: 'foo',
-                owned_by: {
-                    name: 'foo'
-                },
-                created_by: {
-                    name: 'foo'
-                },
-                size: '1',
-                permissions: {
-                    can_rename: true
-                }
+    const props = {
+        file: {
+            created_at: 'foo',
+            description: 'foo',
+            modified_at: 'foo',
+            owned_by: {
+                name: 'foo'
+            },
+            created_by: {
+                name: 'foo'
             },
-            onDescriptionChange: jest.fn(),
-            intl: {
-                locale: 'en'
+            size: '1',
+            permissions: {
+                can_rename: true
             }
-        };
+        },
+        onDescriptionChange: jest.fn(),
+        intl: {
+            locale: 'en'
+        }
+    };
+    test('should render the versions when total_count > 0', () => {
```

Comment:
I think this was my fault, but can you correct the test name?

Reply:
Updated, thanks